src/http-schema/http-schema.ts

```diff
@@ -113,7 +120,7 @@ const PeerReviewSchema = Joi.object({
 });
 
 const AddressSchema = Joi.object({
-  addressCountry: Joi.string().required(),
+  addressCountry: Joi.string().optional(),
```

Comment:
We need to amend our types to reflect the optional status of this and some other properties

Reply:
done